src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.queue.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
+    } catch (e) {
+      console.warn('Error processing command', 'aol-hb.js', e);
```

Comment:
I had the same issue with understanding this, see my previous comment :D But the idea is that this method is overridden after the array is already populated.